build.sbt

```diff
@@ -32,21 +34,21 @@ lazy val coverageSettings: Seq[Setting[_]] = {
 }
 
 lazy val microservice = Project(appName, file("."))
-  .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
+  .enablePlugins(PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
   .disablePlugins(sbt.plugins.JUnitXmlReportPlugin)
   .disablePlugins(JUnitXmlReportPlugin)
   .settings(PlayKeys.playDefaultPort := 9304)
   .settings(
-    majorVersion := 0,
-    scalaVersion := "2.13.10",
     libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test,
-    // To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
-    libraryDependencySchemes ++= Seq("org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always),
-    scalacOptions += "-Wconf:src=routes/.*:s"
+    scalacOptions += "-Wconf:cat=unused-imports&src=routes/.*:s"
   )
-  .configs(IntegrationTest extend Test)
-  .settings(integrationTestSettings(): _*)
+  .configs(Test)
   .settings(resolvers += Resolver.jcenterRepo)
   .settings(coverageSettings: _*)
```

Comment:
Can you please change this to:
`.settings(coverageSettings *)`
Just to stop the IDE syntax warning.